### DISCUSSION TITLE
How to assign your DNS server for a specific client and not filter his requests?

### DISCUSSION THREAD

**Discussion by fizsef** [AdguardTeam/AdGuardHome#6466] [Q&A]
Friends, help. I have a TV BOX in my local network with an IP address of 192.168.0.202. How to make sure that his DNS requests are processed by another DNS server and that his requests are not processed by filters. i.e., that the specified client(192.168.0.202) is not processed by AdguarHome at all. Description on the page https://github.com/AdguardTeam/AdGuardHome/wiki I read it, but I didn't understand and didn't figure out how to do it (

**Comment by Dynamic5912** [comment]
> Friends, help. I have a TV BOX in my local network with an IP address of 192.168.0.202. How to make sure that his DNS requests are processed by another DNS server and that his requests are not processed by filters. i.e., that the specified client(192.168.0.202) is not processed by AdguarHome at all. Description on the page https://github.com/AdguardTeam/AdGuardHome/wiki I read it, but I didn't understand and didn't figure out how to do it (

Add the device as a persistent client by its IP address. Then disable any filtering etc. in the client settings.
